src/Workspaces/Core/Portable/Workspace/Workspace.cs

```diff
@@ -780,49 +780,6 @@ protected virtual void CheckDocumentCanBeRemoved(DocumentId documentId)
         {
         }
 
-        /// <summary>
-        /// Call this method when the text of a document is changed on disk.
-        /// </summary>
-        protected internal void OnDocumentTextLoaderChanged(DocumentId documentId, TextLoader loader)
-        {
-            SetCurrentSolution(
-                oldSolution =>
-                {
-                    CheckDocumentIsInSolution(oldSolution, documentId);
-                    return oldSolution.WithDocumentTextLoader(documentId, loader, PreservationMode.PreserveValue);
-                },
-                WorkspaceChangeKind.DocumentChanged, documentId: documentId,
-                onAfterUpdate: (_, newSolution) => this.OnDocumentTextChanged(newSolution.GetRequiredDocument(documentId)));
```

Comment:
note that the old code here would only update a single document with the loader.  This is in constrast to what we do when setting the SourceText for a document (where we update all links).  This now brings this code in sync with that pattern.